app/engine/rest_handler.rb

```diff
@@ -1,13 +1,16 @@
 module Engine
   class RestHandler < TurnHandler
 
-    MAX_AMMO = 10
+    attr_reader :skew
 
-    def initialize(robot)
+    def initialize(robot, value, skew)
```

Comment:
what does `value` do here? Or is it just there to conform to some generic turn handler interface?

Reply:
Yes, allows for turns to have access to the raw string.  Although I never really use it.  But all the other turns have it too.